pkg/k8s/cnp.go

```diff
@@ -592,26 +592,6 @@ func updateStatusesByCapabilities(client clientset.Interface, capabilities k8sve
 				}
 			}
 		}
-	case capabilities.UpdateStatus:
-		if cnp == nil {
-			return fmt.Errorf("cannot update status of nil CNP")
-		}
-		// k8s < 1.13 has minimal support for JSON patch where kube-apiserver
-		// can print Error messages and even panic in k8s < 1.10.
-		for nodeName, cnpns := range nodeStatuses {
-			cnp.SetPolicyStatus(nodeName, cnpns)
-		}
```

Comment:
Noticed one more thing. 

I see the comment mentions K8s 1.13. Now that with this PR the min version of K8s is 1.12, is this still needed? I think this is a good example of documenting the Capabilities, what they mean, and what version(s) are relevant.

Reply:
I also got confused when I was removing this code. The comment is wrong, as you can see the code is not doing anything related with JSON patch.